### PR TITLE
Add oidc auth to cli clients communicating with pnc

### PR DESCRIPTION
**Pull request by jomrazek**

### CODE REVIEW

**Comment by jomrazek** [comment]
once we're ready also https://github.com/project-ncl/sbomer/blob/main/service/src/main/resources/application-prod.yaml#L52 will need to be reconfigured to service account

**Review by thescouser89 — APPROVED** [review]
LGTM

**Comment by goldmann** [comment]
> LGTM. Does sbomer perform write / delete actions on PNC that would require the token though?

We only read content from PNC API.

**Comment by goldmann** [comment]
@vibe13 Could you please look at this PR?

**Comment by jomrazek** [comment]
I understood it probably like future-proofing, but it is there so we converted it. Also I believe Andrea is on meetings this week.

File: cli/src/main/resources/application.yaml [hunk 1]
```diff
@@ -41,6 +41,12 @@ quarkus:
     console:
       format: "%d{HH:mm:ss,SSS} %-5p [%c{3.}] (%t) %s%e mdc:[%X]%n"
       stderr: true
+  oidc-client:
+    enabled: true
+    auth-server-url: http://localhost:8180/auth/realms/quarkus
```
Comment (vibe13) [inline cli/src/main/resources/application.yaml:46]: @jomrazek This needs to be configured to be our internal SSO right? We have private files for this, but double checking it needs to be configured. Thanks!

Comment (jomrazek) [reply cli/src/main/resources/application.yaml:46]: Yes on internal config files this will be configured to RedHat internal SSO system

**Review by vibe13 — APPROVED** [review]

**Pull request merged**